app/src/main/res/values/styles.xml

```diff
@@ -119,4 +119,10 @@
     <style name="NoConnectivityWarning">
         <item name="android:background">@color/gray_73_percent</item>
     </style>
+    <resources>
+        <style name="AutofillHighlightAppearance" parent="android:autofillHighlight">
+            <item name="android:autofilledHighlight" tools:targetApi="o">@drawable/autofill_item</item>
+        </style>
+    </resources>
```

Comment:
Oh, does this work?

If not, I'd suggest making a `values-26/styles.xml` file, and putting it in there.